extensions-contrib/materialized-view-maintenance/src/test/java/org/apache/druid/indexing/materializedview/MaterializedViewSupervisorTest.java

```diff
@@ -420,4 +360,36 @@ public void testResetOffsetsNotSupported()
         () -> supervisor.resetOffsets(null)
     );
   }
+
+  private List<DataSegment> createBaseSegments()
+  {
+    return Arrays.asList(
+        createSegment("base", "2015-01-01T00Z/2015-01-02T00Z", "2015-01-02"),
```

Comment:
Maybe save the `base` datasource name in a constant as it's used in quite a few places.